dist/t/spec/features/0010_authentication_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "spec_helper"
 
-RSpec.describe "Authentication" do
+RSpec.describe "Authentication", type: :feature do
```

Comment:
If you mark all anyway, can you just remove the condition to the include?

https://github.com/openSUSE/open-build-service/blob/master/dist/t/spec/support/wait_for_ajax.rb#L17

Reply:
Done!